bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WprimeToTauNu_LO/WprimeToTauNu_M-600_kR-0p0001_LO_NNPDF31nnlo/WprimeToTauNu_M-600_kR-0p0001_LO_NNPDF31nnlo_run_card.dat

```diff
@@ -0,0 +1,284 @@
+ 0.0   = ht4min ! minimum Ht for the four leading jets
+ -1.0  = ht2max ! maximum Ht for the two leading jets
+ -1.0  = ht3max ! maximum Ht for the three leading jets
+ -1.0  = ht4max ! maximum Ht for the four leading jets
+#***********************************************************************
+# Photon-isolation cuts, according to hep-ph/9801442                   *
+# When ptgmin=0, all the other parameters are ignored                  *
+# When ptgmin>0, pta and draj are not going to be used                 *
+#***********************************************************************
+ 0.0 = ptgmin ! Min photon transverse momentum
+ 0.4 = R0gamma ! Radius of isolation code
+ 1.0 = xn ! n parameter of eq.(3.4) in hep-ph/9801442
+ 1.0 = epsgamma ! epsilon_gamma parameter of eq.(3.4) in hep-ph/9801442
+ True = isoEM ! isolate photons from EM energy (photons and leptons)
+#*********************************************************************
+# WBF cuts                                                           *
+#*********************************************************************
+ 0.0   = xetamin ! minimum rapidity for two jets in the WBF case  
+ 0.0   = deltaeta ! minimum rapidity for two jets in the WBF case 
+#***********************************************************************
+# Turn on either the ktdurham or ptlund cut to activate                *
+# CKKW(L) merging with Pythia8 [arXiv:1410.3012, arXiv:1109.4829]      *
+#***********************************************************************
+ -1.0  =  ktdurham        
+ 0.4   =  dparameter
+ -1.0  =  ptlund
+ 1, 2, 3, 4, 5, 6, 21  =  pdgs_for_merging_cut ! PDGs for two cuts above   
+#*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+ 4 = maxjetflavor    ! Maximum jet pdg code
+#*********************************************************************
+#
+#*********************************************************************
+# Store info for systematics studies                                 *
+# WARNING: Do not use for interference type of computation           *
+#*********************************************************************
+   True  = use_syst      ! Enable systematics studies
+#
+#**************************************
+# Parameter of the systematics study
+#  will be used by SysCalc (if installed)
+#**************************************                                  
+#
+0.5 1 2 = sys_scalefact  # factorization/renormalization scale factor
+None = sys_alpsfact  # alpha_s emission scale factors
+auto = sys_matchscale # variation of merging scale
+# PDF sets and number of members (0 or none for all members).
+NNPDF23_lo_as_0130_qed = sys_pdf # list of pdf sets
+# MSTW2008nlo68cl.LHgrid 1  = sys_pdf
+#
+
```

Comment:
line 272 to 283 could be removed

Reply:
Hi @menglu21,
Thank you. It's done, I removed these lines. Should I proceed to creation of new gridpacks with these cards or is there something else that needs to be edited?

Reply:
Hi @valesark thanks, i don't have further comment, let's see whether others are fine with them, if not we can merge and you can start to produce gps. @Saptaparna @GurpreetSinghChahal @mlizzo please have a check and feel free to merge